examples/favorites_advanced/redux_favorites_advanced/lib/feature_home/views/home_page.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:redux/redux.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:curved_navigation_bar/curved_navigation_bar.dart';
+
+import 'package:favorites_advanced_base/extensions.dart';
+
+import '../../base/ui_components/puppies_app_bar.dart';
+
+import '../../feature_puppy/favorites/views/favorites_view.dart';
+import '../../feature_puppy/search/views/search_view.dart';
+
+import '../models/navigation_state.dart';
+import '../redux/actions.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage(this.store);
+
+  final Store<NavigationState> store;
+
+  @override
+  Widget build(BuildContext context) => StoreProvider<NavigationState>(
+        store: store,
+        child: Scaffold(
+          appBar: PuppiesAppBar(),
+          bottomNavigationBar: CurvedNavigationBar(
```

Comment:
@GoranPrime, should not you wrap the `CurvedNavigationBar` with `StoreBuilder` so you can react to changes? The same way you have implemented the `body`.

Reply:
Since `StoreBuilder` rebuilds itself every single time the app state changes, it is recommended to use it as little as possible, so I believe this is the best approach.